gateway/ln-gateway/src/federation_manager.rs

```diff
@@ -116,4 +116,20 @@ impl FederationManager {
         // lambda and pass it into `client.with_sync`.
         clients.get(federation_id).cloned()
     }
+
+    // TODO(tvolk131): Optimize this function by adding a reverse map from
+    // federation_id to scid.
```

Comment:
Might not be worth it, this map is likely to be small so I don't think performance is an issue here.

Reply:
Removed.